tests/test_vrf.rb

```diff
@@ -84,11 +84,15 @@ def test_shutdown
     end
     v.shutdown = true
     assert(v.shutdown)
+
+    sleep 20 if node.product_id[/N(5|6)/]
```

Comment:
Ouch. Is this a consistent amount of time or would we be better to have a retry loop here?